partiql-cli/src/main/kotlin/org/partiql/cli/Main.kt

```diff
@@ -237,7 +247,42 @@ internal class MainCommand : Runnable {
             // val connector = LocalPlugin.create(root.toPath())
             // return mapOf("default" to connector)
         }
+
+        if (env != null) {
+            return listOf(parsePartiQL(env!!))
+        }
+
         // Derive a `default catalog from stdin (or file streams)
+        return listOf(parseIon())
+    }
+
+    private fun parsePartiQL(env: File): Catalog {
+        val stream = env.inputStream()
```

Comment:
We should do some validation that the file path is valid and give a more detailed error message. With how it's currently implemented, an invalid file path will print out the entire stack trace.